### PR TITLE
replace vault to local

### PR DESCRIPTION
**Pull request by dntbeSris**
Signed-off-by: zjx244729 <zjx244729@alibaba-inc.com>

# Description

Replace the Secret Store from Hashicorp Vault to Local File

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #667 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -87,10 +87,6 @@ jobs:
       run: |
         docker-compose -f ./sdk-tests/deploy/local-test-kafka.yml up -d
         docker ps
-    - name: Install Local Hashicorp Vault using docker-compose
```
Comment (mukundansundar) [inline .github/workflows/build.yml:90]: have the vault install in the build.yaml and remove it from the validate.yaml file ....

**Review by mukundansundar — CHANGES_REQUESTED** [review]
@dntbeSris One change ... The vault install is removed from the validate.yaml file. 
Because it was removed from build.yaml it is breaking the build.

File: .github/workflows/build.yml (hunk 1 above)
Comment (dntbeSris) [reply .github/workflows/build.yml:90]: ok, and the Integration Test was fail because the vault secret test case. should i fix it ?

Comment (mukundansundar) [reply .github/workflows/build.yml:90]: Yes please @dntbeSris 
cc @artursouza

Comment (dntbeSris) [reply .github/workflows/build.yml:90]: done

**Review by mukundansundar — CHANGES_REQUESTED** [review]
in the integration tests why have the secrets file be created on the file. I think the test local secrets file can be committed as a file in the components directory itself ... 
This is currently causing an issue with other tests also since the component in being loaded in other tests also and it is not finding the secrets file which is only created on the fly for this test ... so dapr initialising is failing because the component is not being loaded ... 

Changing it to a file that exists in the components directory from start is fine ...

**Comment by dntbeSris** [comment]
> 

yep! i create and delete the secret file just in secret test program at first.but it made the other tests failed . because it always load all components . so i created the file earlier. this change in commit [9f21553](https://github.com/dapr/java-sdk/pull/719/commits/9f215534d19dd0a2e34dd5fe8ba158624ab8aac0)

**Comment by dntbeSris** [comment]
> in the integration tests why have the secrets file be created on the file. I think the test local secrets file can be committed as a file in the components directory itself ... This is currently causing an issue with other tests also since the component in being loaded in other tests also and it is not finding the secrets file which is only created on the fly for this test ... so dapr initialising is failing because the component is not being loaded ...
> 
> Changing it to a file that exists in the components directory from start is fine for integration tests ...

what should i do next ？

**Pull request merged**